pkg/storage/mysql/mysqlstore.go

```diff
@@ -333,6 +334,22 @@ func (s *sqlDBStore) Query(findQuery string) (storage.ResultsIterator, error) {
 	return &sqlDBResultsIterator{resultRows: resultRows}, nil
 }
 
+func (s *sqlDBStore) Delete(k string) error {
+	if k == "" {
+		return storage.ErrKeyRequired
+	}
+
+	//nolint: gosec
+	// TODO address SQL injection warning #38
+	_, err := s.db.Exec("DELETE FROM `"+s.tableName+"` WHERE `key`= ?", k)
```

Comment:
the result contains the number of rows affected. Should throw error if this number == 0

Reply:
Done